Assets/Scripts/Player.cs

```diff
@@ -6,6 +7,7 @@ public class Player : MonoBehaviour
     [SerializeField] float runSpeed = 10f;
     [SerializeField] float jumpSpeed = 15f;
     [SerializeField] float climingSpeed = 9f;
```

Comment:
*should be clim**b**ingSpeed ?

Reply:
I understand the concepts in Player.cs in a general way (which function does what), but some tools or solutions are completely new. Anyway, looks cool. :D

Reply:
I have changed the name to climbingSpeed.